services/ingestion/etl/agri/csv/serverless.yml

```diff
@@ -72,12 +72,11 @@ resources:
           Version: '2012-10-17'
           Statement:
             -
-              Sid: AllowAnyoneToPingSNSTopicAGRICSV
+              Sid: AllowIngestionManagerToPushNotificationEffect
```

Comment:
It's not only the ingestion manager which is allowed to **publish** (not push) but everyone/everything/every service.

If we have to update this, I'd rather add the permission on a per-lambda basis (https://stackoverflow.com/a/31493841). No more global "allow everybody to publish". 😉